config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml

```diff
@@ -2771,9 +2771,6 @@
     </details>
   </checkerFrameworkError>
 
-  <!-- This is not possible to fix because if we initialize this in constructor or in
-       field itself pitest will be failed. This is conflict with pitest but value of
-       pitest is above checker -->
```

Comment:
Why are we removing these comments?

Reply:
restored, checker groovy script execution on local removes them when generate patch.